src/main/java/uk/sky/cirrus/locking/CassandraLockConfig.java

```diff
@@ -0,0 +1,117 @@
+
+            case NetworkTopologyStrategy:
+                StringBuilder replicationString = new StringBuilder("'class': 'NetworkTopologyStrategy', ");
+
+                String delimiter = "";
+                for (Map.Entry<String, Integer> dataCenter : dataCenters.entrySet()) {
+                    replicationString
+                            .append(delimiter)
+                            .append("'")
+                            .append(dataCenter.getKey())
+                            .append("': ")
+                            .append(dataCenter.getValue());
+
+                    delimiter = ", ";
+                }
+
+                return replicationString.toString();
+
+            default:
+                throw new IllegalArgumentException("Unexpected replication class: " + replicationClass);
+        }
+    }
+
+    public static CassandraLockConfigBuilder builder() {
+        return new CassandraLockConfigBuilder();
+    }
+
+    public static class CassandraLockConfigBuilder extends LockConfig.LockConfigBuilder {
+
+        private int replicationFactor = 1;
+        private ReplicationClass replicationClass = ReplicationClass.SimpleStrategy;
+        private Map<String, Integer> dataCenters = new HashMap<>();
+
+        private CassandraLockConfigBuilder() {}
+
+        /**
+         * Sets the replication class to SimpleStrategy and replication factor to {@code replicationFactor}.
+         *
+         * @param replicationFactor defaults to 1
+         * @return this
+         * @throws IllegalArgumentException if data centers have been configured using {@link #withNetworkTopologyReplication(String, int)}
+         */
+        public CassandraLockConfigBuilder withSimpleStrategyReplication(int replicationFactor) {
+            if (!dataCenters.isEmpty())
+                throw new IllegalArgumentException("Replication class 'SimpleStrategy' cannot be used with data centers: " + dataCenters);
+
+            this.replicationFactor = replicationFactor;
+            this.replicationClass = ReplicationClass.SimpleStrategy;
+            return this;
+        }
+
+        /**
+         * Sets the replication class to NetworkTopologyStrategy and replication factor for each {@code dataCenter} to {@code replicationFactor}.
+         * Can be called multiple times to add more data centers.
+         * Default is SimpleStrategy with replication factor of 1.
+         *
+         * @param dataCenter
+         * @param replicationFactor
+         * @return this
+         */
+        public CassandraLockConfigBuilder withNetworkTopologyReplication(String dataCenter, int replicationFactor) {
+            dataCenters.put(dataCenter, replicationFactor);
+            this.replicationClass = ReplicationClass.NetworkTopologyStrategy;
+            return this;
+        }
+
+        @Override
+        public CassandraLockConfigBuilder withPollingInterval(Duration pollingInterval) {
```

Comment:
Does this need to be configurable? Can't we pick a value that should work for everyone?